test/metabase/driver/query_processor/expand_resolve_test.clj

```diff
@@ -11,82 +11,79 @@
+   {:database     {:name    "test-data"
+                   :details {:short-lived? nil
+                             :db           "mem:test-data;USER=GUEST;PASSWORD=guest"}
+                   :id      (id)
+                   :engine  :h2}
+    :type         :query
+    :query        {:source-table {:schema "PUBLIC"
+                                  :name   "VENUES"
+                                  :id     (id :venues)}
+                   :filter       {:filter-type :>
+                                  :field       {:field-id           (id :venues :price)
+                                                :field-name         "PRICE"
+                                                :field-display-name "Price"
+                                                :base-type          :IntegerField
+                                                :special-type       :category
+                                                :visibility-type    :normal
+                                                :table-id           (id :venues)
+                                                :schema-name        "PUBLIC"
+                                                :table-name         "VENUES"
+                                                :position           nil
+                                                :description        nil
+                                                :parent-id          nil
+                                                :parent             nil}
+                                  :value       {:value 1
+                                                :field {:field-id           (id :venues :price)
+                                                        :field-name         "PRICE"
+                                                        :field-display-name "Price"
+                                                        :base-type          :IntegerField
+                                                        :special-type       :category
+                                                        :visibility-type    :normal
+                                                        :table-id           (id :venues)
+                                                        :schema-name        "PUBLIC"
+                                                        :table-name         "VENUES"
+                                                        :position           nil
+                                                        :description        nil
+                                                        :parent-id          nil
+                                                        :parent             nil}}}
+                   :join-tables  nil}
+    :fk-field-ids #{}
+    :table-ids    #{(id :venues)}}]
+  (let [expanded-form (ql/expand (ql/wrap-inner-query (query venues
+                                                        (ql/filter (ql/and (ql/> $price 1))))))]
```

Comment:
@camsaul , I'm curious why you modified these unit tests when I had just created them and explained that I intentionally had written out the query manually ... https://github.com/metabase/metabase/pull/2353#discussion_r59311779